### ISSUE TITLE
[![Mend Renovate](https://app.renovatebot.com/images/banner.svg)](https://renovatebot.com)

### ISSUE THREAD

**Issue by Giovanni0517** [Giovanni0517/Tara-#2]
[![Mend Renovate](https://app.renovatebot.com/images/banner.svg)](https://renovatebot.com)

Welcome to [Renovate](https://togithub.com/renovatebot/renovate)! This is an onboarding PR to help you understand and configure settings before regular Pull Requests begin.

🚦 To activate Renovate, merge this Pull Request. To disable Renovate, simply close this Pull Request unmerged.

---
### Detected Package Files

 * `package.json` (npm)

### Configuration Summary

Based on the default config's presets, Renovate will:

  - Start dependency updates only once this onboarding PR is merged
  - Enable Renovate Dependency Dashboard creation.
  - If Renovate detects semantic commits, it will use semantic commit type `fix` for dependencies and `chore` for all others.
  - Ignore `node_modules`, `bower_components`, `vendor` and various test/tests directories.
  - Group known monorepo packages together.
  - Use curated list of recommended non-monorepo package groupings.
  - `apollo-server` packages became scoped.
  - `babel-eslint` was renamed under the `@babel` scope.
  - Replace containerbase dependencies.
  - `cucumber` became scoped.
  - `fastify` packages became scoped.
  - `hapi` became scoped.
  - Jade was renamed to Pug.
  - `joi` became scoped under the `hapi` organization.
  - `joi` was moved out of the `hapi` organization.
  - The Kubernetes container registry has changed from `k8s.gcr.io` to `registry.k8s.io`.
  - The Kubernetes container registry has changed from `k8s.gcr.io` to `registry.k8s.io`.
  - The `material-ui` monorepo org was renamed from `@material-ui` to `@mui`.
  - The `messageformat` monorepo package naming scheme changed from `messageFormat-{{package}}`-to-`@messageformat/{{package}}`.
  - `middie` became scoped.
  - `now` was renamed to `vercel`.
  - `@parcel/css` was renamed to `lightningcss`.
  - `react-query/devtools` became scoped under the `tanstack` organization.
  - `react-query` became scoped under the `tanstack` organization.
  - `react-scripts` supports TypeScript since version `2.1.0`.
  - The `@renovate/pep440` package was renamed to `@renovatebot/pep440`.
  - The node-resolve plugin for rollup became scoped.
  - The `vso-task-lib` package is now published as `azure-pipelines-task-lib`.
  - The `vsts-task-lib` package is now published as `azure-pipelines-task-lib`.
  - The `xmldom` package is now published as `@xmldom/xmldom`.
  - A collection of workarounds for known problems with packages.

🔡 Would you like to change the way Renovate is upgrading your dependencies? Simply edit the `renovate.json` in this branch with your custom config and the list of Pull Requests in the "What to Expect" section below will be updated the next time Renovate runs.

---

### What to Expect

With your current configuration, Renovate will create 2 Pull Requests:

<details>
<summary>Update dependency web-vitals to v3.3.0</summary>

  - Schedule: ["at any time"]
  - Branch name: `renovate/web-vitals-3.x-lockfile`
  - Merge into: `main`
  - Upgrade [web-vitals](https://togithub.com/GoogleChrome/web-vitals) to `3.3.0`

</details>

<details>
<summary>Update dependency @testing-library/react to v14</summary>

  - Schedule: ["at any time"]
  - Branch name: `renovate/testing-library-react-14.x`
  - Merge into: `main`
  - Upgrade [@testing-library/react](https://togithub.com/testing-library/react-testing-library) to `^14.0.0`

</details>

---

❓ Got questions? Check out Renovate's [Docs](https://docs.renovatebot.com/), particularly the Getting Started section.
If you need any further assistance then you can also [request help here](https://togithub.com/renovatebot/renovate/discussions).

---

This PR has been generated by [Mend Renovate](https://www.mend.io/free-developer-tools/renovate/). View repository job log [here](https://app.renovatebot.com/dashboard#github/Giovanni0517/codespaces-react).

_Originally posted by @renovate in https://github.com/Giovanni0517/codespaces-react/pull/1_
